CodingChallenges/CC_113_Hypercube/P5/P4Vector.js

```diff
@@ -0,0 +1,27 @@
+// Daniel Shiffman
+// http://youtube.com/thecodingtrain
+// http://codingtra.in
+// Javascript transcription: Chuck England
```

Comment:
```suggestion
// JavaScript transcription: Chuck England
```
Here is the last one. Then we can merge :D